From 4524c1e330342f12c58f2de1d3b1158a80b097d6 Mon Sep 17 00:00:00 2001
From: Hoa Nguyen <hoanguyen@ucdavis.edu>
Date: Thu, 2 Jun 2022 21:25:25 +0000
Subject: [PATCH 681/757] mem-ruby: Replace the deprecated Stats namespace

Using the old namespace generates quite a few warnings during
compilation time.

Change-Id: Ibbb8f54f0bee974443dc5dfcc30e95100b8c24d6
Signed-off-by: Hoa Nguyen <hoanguyen@ucdavis.edu>
Reviewed-on: https://gem5-review.googlesource.com/c/public/gem5/+/60289
Tested-by: kokoro <noreply+kokoro@google.com>
Reviewed-by: Giacomo Travaglini <giacomo.travaglini@arm.com>
Maintainer: Bobby Bruce <bbruce@ucdavis.edu>
Reviewed-by: Jason Lowe-Power <power.jg@gmail.com>
---
 src/mem/ruby/structures/MN_TBEStorage.hh | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/mem/ruby/structures/MN_TBEStorage.hh b/src/mem/ruby/structures/MN_TBEStorage.hh
index e314e1f..1adbcca 100644
--- a/src/mem/ruby/structures/MN_TBEStorage.hh
+++ b/src/mem/ruby/structures/MN_TBEStorage.hh
@@ -229,19 +229,19 @@ class MN_TBEStorage
     }
 
   private:
-    struct MN_TBEStorageStats : public Stats::Group
+    struct MN_TBEStorageStats : public statistics::Group
     {
-        MN_TBEStorageStats(Stats::Group *parent)
-          : Stats::Group(parent),
+        MN_TBEStorageStats(statistics::Group *parent)
+          : statistics::Group(parent),
             ADD_STAT(avg_size, "Avg. number of slots allocated"),
             ADD_STAT(avg_util, "Avg. utilization"),
             ADD_STAT(avg_reserved, "Avg. number of slots reserved")
         {}
 
         // Statistical variables
-        Stats::Average avg_size;
-        Stats::Average avg_util;
-        Stats::Average avg_reserved;
+        statistics::Average avg_size;
+        statistics::Average avg_util;
+        statistics::Average avg_reserved;
     } m_stats;
 
     std::vector<TBEStorage *> partitions;
-- 
1.8.3.1

